resources/measures/HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -2706,6 +2706,18 @@ def self.apply_heating_setpoints(model, runner, weather, htg_wkdy_monthly, htg_w
       end
       break # assume all finished zones have the same schedules
     end
+    
+    # Store the setpoints on thermostat_setpoint for DR measure
+    finished_zones.each do |finished_zone|
+      thermostat_setpoint = finished_zone.thermostatSetpointDualSetpoint
+      if thermostat_setpoint.is_initialized
+        thermostat_setpoint = thermostat_setpoint.get 
+        thermostat_setpoint.additionalProperties.setFeature("htg_wkdy", "#{htg_wkdy_monthly[0].join(",")}")
```

Comment:
`htg_wkdy_monthly` is a 12-element array of arrays, right? Where each element corresponds to a month of the year? Are you ok with always taking the 24-hour schedule corresponding to January here?

Reply:
I did this because there doesn't seem to be a circumstance in which months would differ from each other. For example, when setting cooling setpoints, the 24-hour schedule is repeated for all months: ` success = HVAC.apply_cooling_setpoints(model, runner, weather, [weekday_setpoints] * 12, [weekend_setpoints] * 12, use_auto_season, season_start_month, season_end_month)`